articles/multifactor-authentication/guardian/user-guide.md

```diff
@@ -23,7 +23,9 @@ Next, a code will appear, you will have five minutes to scan the code before it
 
 ![](/media/articles/mfa/guardian-code.png)
 
-Once your code has been successfully scanned, you will see a confirmation screen which includes a recovery code. If for some reason you do not have your mobile device, you will need this recovery code to login. Make sure to take note of this code and keep it somewhere safe. Check the box that you have recorded the code, and then you are logged in.
+> **Android Users:** When using the Guardian app for Android, there is an additional step of adding a passphrase. This recovery passphrase is only required when the security settings on your device have changed. You can use suggested passphrase or create your own. ![](/media/articles/mfa/android-passphrase.png)
+
+After the code has been successfully scanned, you will see a confirmation screen which includes a recovery code. If for some reason you do not have your mobile device, you will need this recovery code to login. Make sure to take note of this code and keep it somewhere safe. Check the box that you have recorded the code, and then you are logged in.
```

Comment:
@angelaLazar Since the passphrase creation is only required after the first successful enroll, we can move the android specific note below this. And maybe don't be that specific about when the passphrase will be required, only  mention that it might be required in some weird situations (not using that exact words :)). Something like this:

> **Android Users:** When using the Guardian app for Android, after the first enroll you will be required to create a passphrase. This recovery passphrase will not be required every time you use the app, but could be required when some Android security settings have been changed. You can use the suggested passphrase or create your own. ![](/media/articles/mfa/android-passphrase.png)